dspace-server-webapp/src/test/resources/org/dspace/app/rest/openalex-institution-single.json

```diff
@@ -0,0 +1,4426 @@
+{
+  "meta": {
```

Comment:
Same here, this is a very large test file (>4,000 lines).  For all of these test JSON files that are >1,000 lines, I'd like to know if we could trim them down.  This PR is over 27,000 lines of code, but it looks like >21,000 lines are just these very large test files.  So, the PR itself would be much smaller if we could simply trim these files down in size.